app.py

```diff
@@ -39,7 +39,7 @@ def validate():
         prediction_is_correct = (validation_form.is_correct.data ==
                                  validation_form.thumbs_up)
         requests.post(server_url + "/validate",
-                      {"validation": json.dumps(prediction_is_correct)},
+                      {"validation": json.dumps(prediction_is_correct), "sender": "with-emojis"},
```

Comment:
@kandrio I added a key-value pair to the dictionary sent as request that looks like the following: {"sender": "with-emojis"}